compat/test/browser/suspense.test.js

```diff
@@ -739,8 +739,8 @@ describe('suspense', () => {
 				expect(scratch.innerHTML).to.eql(
 					`<div>Hello first 2</div><div>Hello second 2</div>`
 				);
-				expect(Suspender1.prototype.render).to.have.been.calledThrice;
-				expect(Suspender2.prototype.render).to.have.been.calledThrice;
+				expect(Suspender1.prototype.render).to.have.been.calledTwice;
```

Comment:
With this optimization the `render` is only called twice since the VNode can be skipped since it never changes. `Suspense` does not insert any extra DOM around children

Reply:
Yup. I was thinking that the Suspender component was the one that suspended so it would need to get rerendered in order for the `<div>Hello first 2</div>` VNode to get added to the tree. But in actuality Suspender just renders a `Lazy` component, and that `Lazy` component is the component that actually suspends (i.e. throws the promise). So because of #2125 all of the components that suspended (the Lazy components, not the Suspender components) are queued for updates using `forceUpdate`.

That was the piece I was missing. My mental model had forgotten about the Lazy component under Suspender and was thinking Suspender needed to rerender. So perhaps these test should check is that the Lazy component's render is called after all the promises are resolved, not the Suspender component but that's not in scope of this PR. I'm just talking out loud to share my understanding lol 😅